tool/site/client-react/tests/pages/claimPage.spec.tsx

```diff
@@ -78,18 +80,14 @@ test("Displays an error if dispense-notification not found", async () => {
 
 test("Displays an error on invalid response", async () => {
   moxios.stubRequest(prescriptionOrderUrl, {
-    status: 200,
+    status: 500,
     response: {}
   })
-  moxios.stubRequest(dispenseNotificationUrl, {
-    status: 200,
-    response: [{}]
-  })
 
-  const {container} = render(<ClaimPage baseUrl={baseUrl} prescriptionId={prescriptionId}/>)
+  const {container} = renderWithContext(<ClaimPage prescriptionId={prescriptionId}/>, context)
   await waitFor(() => screen.getByText("Error"))
 
-  expect(screen.getByText("Failed to retrieve prescription details.")).toBeTruthy()
+  expect(screen.getByText("Request failed with status code 500")).toBeTruthy()
```

Comment:
with changes like these, it doesnt really matter what exact text is shown on screen. i wonder if we can check the state of the page component (or its child) to see if it has an error state

Reply:
We're already doing that by expecting the header to be "Error" on line 88. Shall I just remove the assertion?

Reply:
if its not too much hassle :+1: